
import public "MessageID.proto";


message ReqLogin {

    required MessageID  msg_id = 1 [default = MSG_REQ_LOGIN];
    required string username        = 2;
    required string password        = 3;
}

enum NOTIFY_LOGIN_RESULT
{
    NLR_SUCCESS = 0;    // 成功
    NLR_FAILED  = 1;    // 失败
}

// 通知登陆认证结果
message NotifyLoginResult {
    
    required MessageID  msg_id = 1 [default = MSG_REQ_LOGIN];
    required NOTIFY_LOGIN_RESULT    result = 2;
}
